app/javascript/components/subject/CaseStatus.js

```diff
@@ -132,7 +132,36 @@ class CaseStatus extends React.Component {
   }
 
   createModal(title, toggle, submit) {
-    if (this.state.case_status === '') {
+    if (!this.props.patient.monitoring) {
+      return (
+        <Modal size="lg" show centered onHide={toggle}>
+          <Modal.Header>
+            <Modal.Title>{title}</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <p>
+              Are you sure you want to change case status from {this.props.patient.case_status} to {this.state.case_status}? Since this record is on the Closed
```

Comment:
The only difference between this modal and the usual modal when the record is not closed is the actual prompt language. Would ask that you make a method that creates this modal and adjust the language just based on the monitoring status to avoid code duplication.

Reply:
@kylietmo my refactor would help with this, do you want me to push it to this branch? it includes the changes for [JIRA 875](https://tracker.codev.mitre.org/browse/SARAALERT-875)